src/status.h

```diff
@@ -41,6 +41,8 @@ struct status_main {
 	int compat;
 	int pass;
 	int progress;
+	int resume_salt;
+	uint32_t *resume_salt_md5;
```

Comment:
Is the status struct really the correct place for this? The database seems like a better place for it.